Tests/TuistSupportIntegrationTests/System/SystemIntegrationTests.swift

```diff
@@ -30,7 +30,7 @@ final class SystemIntegrationTests: TuistTestCase {
 
     func test_run_output_is_redirected() throws {
         var output = ""
-        try subject.runAndPrint(
+        try subject.run(
```

Comment:
Can we add an assertion the output is not printed?

Reply:
I dont know that we can easily.  If we pass `.none` or `.collect` the output ends up elsewhere and we dont have a thing to test for.  These interfaces could use a refactor to better align them with what is provided by `Process`.  e.g. even if we passed back the `Result` here, i dont think it would correctly contain the output or error data if we passed `.collect` into this method since we're still passing `.stream` into the `Process` initializer.

Reply:
gotcha, we can skip adding a test for now then 👍 

Yeah, it sounds a refactor would simplify things for sure